presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/view/ImageDialogView.java

```diff
@@ -0,0 +1,61 @@
+package com.globant.equattrocchio.cleanarchitecture.mvp.view;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.app.DialogFragment;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.bumptech.glide.Glide;
+import com.globant.equattrocchio.cleanarchitecture.R;
+import com.globant.equattrocchio.cleanarchitecture.util.bus.RxBus;
+import com.globant.equattrocchio.cleanarchitecture.util.bus.observers.DeleteClickedObserver;
+import com.globant.equattrocchio.domain.model.CompleteImage;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+public class ImageDialogView {
```

Comment:
Create a new class similar to ActivityView, to use with Fragment instead of Activity. And avoid mantain the ref DialogFragment without WekRef